### ISSUE TITLE
Switching to Neural View (F2) leads to crash

### ISSUE THREAD

**Issue by creiser** [darglein/ADOP#11]
Switching to Neural View (F2) leads to crash on all scenes except of the boat scene:
```
(adop) creiser@creiser-home:~/ADOP$ ./build/bin/adop_viewer --scene_dir scenes/tt_lighthouse/
register neural render info
Ref. 
=============== Saiga ===============
| Saiga Version     1.3.2           |
| Eigen Version     3.3.91          |
| Compiler          GNU             |
|   -> Version      9.3.0           |
| Build Type        RelWithDebInfo  |
| Debug             0               |
| Eigen Debug       0               |
| ASAN              0               |
| Asserts           1               |
| Optimizations     0               |
=====================================
Initializing GLFW.
Initializing GLFW sucessfull!
Creating GLFW Window. 2560x1440 Mode=3 Fullscreen=1 Borderless=0
=========================== OpenGL ===========================
| OpenGL Version    3.2.0 NVIDIA 470.74                      | 
| GLSL Version      1.50 NVIDIA via Cg compiler              | 
| Renderer          NVIDIA GeForce GTX 1080 Ti/PCIe/SSE2     | 
| Vendor            NVIDIA Corporation                       | 
==============================================================
[Renderer] Target resized to 2488x1369
loading shader /home/creiser/ADOP/External/saiga/shader/imgui_gl.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/tone_map.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/tone_map_linear.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/bloom_extract_bright.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/bloom_downsample.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/bloom_upsample.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/bloom_combine_simple.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/copy_image.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/compute_blur.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/compute_blur.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/lighting/light_directional.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/lighting/light_directional.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/lighting/light_point.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/lighting/light_point.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/lighting/light_spot.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/lighting/light_spot.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/lighting/light_point.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/lighting/light_spot.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/lighting/stenciltest.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/post_processing/post_processing.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/lighting/blitDepth.glsl
Deferred Renderer initialized. Render resolution: 2488x1369
loading shader /home/creiser/ADOP/External/saiga/shader/post_processing/imagedisplay.glsl
Program Initialized!
Loading Scene scenes/tt_lighthouse/
====================================
Scene Loaded
  Name       
  Path       scenes/tt_lighthouse/
  Image Size 2048x1080
  Aspect     1.8963
  K          1666.68 1661.18 1024    540     0      
  ocam       2048x1080 affine(1, 0, 0, 0, 0) cam2world() world2cam()
  ocam cut   1
  normalized center 0 0
  dist       -0.126731   0.0217503   0           0           0           0           0.000406465 0.000123456
  Points     12313620
  Colors     12313620
  Num Images 309
  Num Cameras 1
====================================
loading shader /home/creiser/ADOP/External/saiga/shader/asset/ColoredAsset.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/asset/ColoredAsset.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/asset/ColoredAsset.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/asset/LineVertexColoredAsset.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/asset/LineVertexColoredAsset.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/asset/LineVertexColoredAsset.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/tone_map.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/tone_map_linear.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/bloom_extract_bright.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/bloom_downsample.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/bloom_upsample.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/bloom_combine_simple.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/copy_image.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/compute_blur.glsl
loading shader /home/creiser/ADOP/External/saiga/shader/compute/compute_blur.glsl
Found experiment experiments//2021-10-30_09:17:38_multi_scene_pose/ with 1 epochs
Found experiment experiments//2021-10-30_09:14:22_boat/ with 1 epochs
[Renderer] Target resized to 2004x1030
[DeferredRenderer] Resize 2004x1030
loading shader shader//point_render.glsl
terminate called after throwing an instance of 'c10::Error'
  what():  Accessing empty ModuleHolder
Exception raised from get at /home/creiser/miniconda3/envs/adop/lib/python3.9/site-packages/torch/include/torch/csrc/api/include/torch/nn/pimpl.h:108 (most recent call first):
frame #0: c10::Error::Error(c10::SourceLocation, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >) + 0x6c (0x7f610508f7ac in /home/creiser/miniconda3/envs/adop/lib/python3.9/site-packages/torch/lib/libc10.so)
frame #1: c10::detail::torchCheckFail(char const*, char const*, unsigned int, char const*) + 0xf5 (0x7f610505b9b1 in /home/creiser/miniconda3/envs/adop/lib/python3.9/site-packages/torch/lib/libc10.so)
frame #2: RealTimeRenderer::Render(ImageInfo) + 0x143d (0x7f612669bd1d in /home/creiser/ADOP/build/bin/libNeuralPoints.so)
frame #3: <unknown function> + 0x1d847 (0x55fbca82d847 in ./build/bin/adop_viewer)
frame #4: Saiga::DeferredRenderer::renderGL(Saiga::Framebuffer*, Saiga::ViewPort, Saiga::Camera*) + 0xaf5 (0x7f6126322575 in /home/creiser/ADOP/build/External/saiga/lib/libsaiga_opengl.so)
frame #5: Saiga::OpenGLRenderer::render(Saiga::RenderInfo const&) + 0x150 (0x7f612636b430 in /home/creiser/ADOP/build/External/saiga/lib/libsaiga_opengl.so)
frame #6: Saiga::WindowBase::render() + 0x4c (0x7f612577c19c in /home/creiser/ADOP/build/External/saiga/lib/libsaiga_core.so)
frame #7: Saiga::MainLoop::render(float, float) + 0x7a (0x7f612577a87a in /home/creiser/ADOP/build/External/saiga/lib/libsaiga_core.so)
frame #8: Saiga::MainLoop::startMainLoop(Saiga::MainLoopParameters) + 0x25c (0x7f612577ac5c in /home/creiser/ADOP/build/External/saiga/lib/libsaiga_core.so)
frame #9: Saiga::WindowBase::startMainLoop(Saiga::MainLoopParameters) + 0x2b (0x7f612577c5cb in /home/creiser/ADOP/build/External/saiga/lib/libsaiga_core.so)
frame #10: <unknown function> + 0x1945c (0x55fbca82945c in ./build/bin/adop_viewer)
frame #11: __libc_start_main + 0xf3 (0x7f6104ce80b3 in /lib/x86_64-linux-gnu/libc.so.6)
frame #12: <unknown function> + 0x1986e (0x55fbca82986e in ./build/bin/adop_viewer)

Aborted (core dumped)
```

**Comment by darglein** [comment]
Thanks for reporting this issue. It appears that the trailing `/` breaks the file name processing. I will try to fix this issue now. In the mean time use:

`./build/bin/adop_viewer --scene_dir scenes/tt_lighthouse`

**Comment by darglein** [comment]
This issue should be fixed with 51fbf9e1e0acbe2fc051ad353a45480181c3e394

Please verify if all scenes are now working on your system.

**Comment by creiser** [comment]
Yes, works now with a trailing / as well.
